modules/virt-adding-secret-configmap-service-account-to-vm.adoc

```diff
@@ -16,11 +16,11 @@ Add a secret, ConfigMap, or service account to a virtual machine by using the {p
 
```

Comment:
The existing documentation states that the VM must be powered off as a prerequisite to making any changes in the Environment tab. Is that still accurate? If yes, then since the VM is not running there would not be any question of pending changes to be applied after restart.